blueprints/ember-redux-shim/index.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
         return this.addAddonsToProject({
           packages: [
             {name: 'ember-lodash-shim', target: '^2.0.0'},
-            {name: 'ember-symbol-observable', target: '0.1.2'}
+            {name: 'ember-symbol-observable', target: '0.1.2'},
+            {name: 'babel-plugin-transform-object-rest-spread', target: '6.23.0'}
```

Comment:
Another question for you on the subject - say I have an addon that requires ember fetch be installed because I use it during treeForAddon work ... that something that should be a dependency or added to the blueprints ?